src/main/java/seedu/recipe/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.recipe.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.recipe.logic.commands.exceptions.CommandException;
+import seedu.recipe.model.Model;
+import seedu.recipe.model.plan.PlannedRecipeWithinDateRangePredicate;
+
+/**
+ * Schedules a recipe to a date.
+ */
+
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the planed recipes in the specified format.";
```

Comment:
It should be "planned" and not "planed" right? :)

Reply:
Yes! Thanks for spotting it